### ISSUE TITLE
Add empty alt text attribute for project hero images 

### ISSUE THREAD

**Issue by roslynwythe** [hackforla/website#4670]
### Overview
As developers we need to markup image attributes according to W3C guidelines in order to ensure accessibility.  Each project page features a hero image which does not add meaning that differs from the adjacent H1 header; therefore the hero images should be marked with an empty alt attribute.  

### Action Items
- [x] Read the file `_layouts/project.html` and locate the `<img>` tag located within `<header class='project-hero'>`
- [x] To the `<img>` tag, add the attribute: `alt=''`

### Resources/Instructions

- https://jekyllrb.com/docs/step-by-step/04-layouts/
- [W3C Alt text Decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/)

**Comment by ShivangRawat30** [comment]
hey, can you assign this to me ?

**Comment by ShivangRawat30** [comment]
#4672

**Comment by roslynwythe** [comment]
Hi @ShivangRawat30 these issues are open only to members of Hack for LA.  You don't have to live in LA to join.  Please see https://hackforla.org/join for more information.

**Comment by Mayank2808sharma** [comment]
Availability: Monday to Friday (9 am to 5 pm)

ETA: completed the task.

**Comment by Adastros** [comment]
Unassigning dev from issue because this issue is still being drafted. Pull request #4727 closed

**Comment by Dprosser4** [comment]
Availability for this week: Weekdays 9:00am - 8:30 PM PST
My estimated ETA for completing this issue: Today
